dominate/util.py

```diff
@@ -184,3 +184,11 @@ def raw(s):
   Inserts a raw string into the DOM. Unsafe. Alias for text(x, escape=False)
   '''
   return text(s, escape=False)
+
+class novalue:
```

Comment:
I'd recommend changing this to `set = ()` so you can do:

```py
tags.a('foo', up_instant=set)
```

IMO, there is no need for it to be a class and be instantiated every time.

Alternatively or additionally, the html spec refers to such attributes as boolean attributes.  So, IMO, the attribute handling code could just look for `is True` and, if that's the case, set the attr name without the value.

```
tags.a('foo', up_instant=True)
```

Personally, I'd just go with supporting `True`, document it, and and not add the special sentinel value for simplicity sake.

Reply:
I just realize boolean attrs are already supported.